DeepFried2/layers/UpSample.py

```diff
@@ -0,0 +1,36 @@
+import DeepFried2 as df
+from DeepFried2.utils import create_param_and_grad
+
+
+class UpSample(df.Module):
+    def __init__(self, upsample=(2,2), output_shape=None):
+        df.Module.__init__(self)
+        self.upsample = upsample
+        self.output_shape = output_shape
+
+    def symb_forward(self, symb_input):
+        """symb_input shape: 2D: (n_input, channels, height, width)
+                             3D: (n_input, channels, depth, height, width)
+
+           upsample determines how many upsampling steps are taken. (height,width) for 2D or (depth,height,width) for 3D
+           output_shape can be used to crop the upsampled result to a desired size. (height,width) for 2D or (depth,height,width) for 3D
```

Comment:
These two lines should be doc-string of the `__init__` method since it describes that one's parameters.